src/Platform/PlatformUnix.cpp

```diff
@@ -16,6 +16,7 @@
  * You should have received a copy of the GNU General Public License
  * along with Falltergeist.  If not, see <http://www.gnu.org/licenses/>.
  */
+#if defined(__unix__) || defined(__APPLE__) || defined(__linux__)
```

Comment:
@alexeevdv, why? The question is precisely if it should. We are guarding inside the file for ignoring it whole. That's CMake's job. If it's a piece of code inside that is platform specific, it makes sense. If it's the whole file platform specific, I don't believe so.